dataplane/api-server/src/lib.rs

```diff
@@ -42,23 +44,37 @@ pub async fn start(
     let healthchecks = tokio::spawn(async move {
         let (_, health_service) = tonic_health::server::health_reporter();
         let mut server_builder = Server::builder();
-        server_builder
+
+        let port = port + 1;
+        let addr = SocketAddrV4::new(addr, port);
+        let server = server_builder
             .add_service(health_service)
-            .serve(SocketAddrV4::new(addr, port + 1).into())
-            .await
-            .unwrap();
+            .serve(addr.into());
+
+        debug!("gRPC Health Checking service listens on port {}", port);
```

Comment:
As a personal preference, I would prefer to see this and the other line log the `{addr}:{port}` together, rather than logging the address, then the ports on separate lines. Thoughts?